setup.py

```diff
@@ -29,7 +29,7 @@
         'scikit-learn>=0.21.2',
         # FIXME: Use stable release instead of tracking master once
         # commit 9a760542 is released.
-        'stable-baselines @ git+https://github.com/hill-a/stable-baselines.git',
+        'stable-baselines @ git+https://github.com/hill-a/stable-baselines.git@vec-normalize-pickle',
```

Comment:
Looks like 9a760542 ready (so should update `FIXME`), but should keep at 'stable-baselines @ git+https://github.com/hill-a/stable-baselines.git' since `vec-normalize-pickle` isn't released yet.